sdk/src/program-manager.ts

```diff
@@ -352,6 +354,7 @@ class ProgramManager {
         recordSearchParams?: RecordSearchParams,
         feeRecord?: string | RecordPlaintext,
         privateKey?: PrivateKey,
+        skipCertificate?: boolean,
```

Comment:
On second thought, let's make these their own methods and ensure the methods take obejcts as inputs. Reasoning is that we definitely want better domain separation and to indicate what methods are for DevNode.